cmd/snap-mgmt/snap-mgmt.sh.in

```diff
@@ -33,7 +33,7 @@ systemctl_stop() {
 }
 
 purge() {
-    # debian-9, fedora-27, ubuntu-16.04...
+    # shellcheck disable=SC1091
     distribution=$(. /etc/os-release; echo "${ID}-${VERSION_ID}")
```

Comment:
Can we just do this:
distribution="$(. /etc/os-release; echo "${ID}-${VERSION_ID}")"

Reply:
```
distribution="$(. /etc/os-release; echo ${ID}-${VERSION_ID})"
```
should be fine for shellcheck

Reply:
It is unhappy about following a source that is outside the source tree, the quoting is ok